packages/expo-build-properties/src/pluginConfig.ts

```diff
@@ -137,6 +156,94 @@ export interface PluginConfigTypeIos {
    * SDK 49+ is required.
    */
   unstable_networkInspector?: boolean;
+
+  /**
+   * Add extra CocoaPods dependencies for all targets.
+   *
+   * This acts like to add the following code to **ios/Podfile**:
+   * ```
+   * pod '[EXTRA_POD_NAME]', '~> [EXTRA_POD_VERSION]'
+   * # e.g.
+   * pod 'Protobuf', '~> 3.14.0'
+   * ```
+   *
+   * @hide For the implementation details,
+   * this property is actually handled by `expo-modules-autolinking` but not the config-plugins inside expo-build-properties.
+   */
+  extraPods?: ExtraIosPodDependency[];
+}
+
+/**
+ * Interface representing extra CocoaPods dependency.
+ * @see https://guides.cocoapods.org/syntax/podfile.html#pod
```

Comment:
```suggestion
 * @see [https://guides.cocoapods.org/syntax/podfile.html#pod](https://guides.cocoapods.org/syntax/podfile.html#pod)
```

Suggestion here to convert it to a link when rendered. Currently its just a text:

<img width="887" alt="CleanShot 2023-06-09 at 00 47 45@2x" src="https://github.com/expo/expo/assets/10234615/fd2cda5b-2d84-4e08-bfa4-a5354ede6bf5">